src/Test/contconic.jl

```diff
@@ -1580,6 +1669,94 @@ powtests = Dict("pow1v" => pow1vtest,
 
 @moitestset pow
 
+function _dualpow1test(model::MOI.ModelLike, config::TestConfig, vecofvars::Bool)
+    atol = config.atol
+    rtol = config.rtol
+    # Problem dual POW1
+    # min -2x_1 - x_2
```

Comment:
If the objective here is `-x_1 - x_2` the problem is easy to solve by hand so it might be easier to debug and we have the analytical solution.
By the [Weighted AM–GM inequality](https://en.wikipedia.org/wiki/Inequality_of_arithmetic_and_geometric_means), you have
0.9a + 0.1b >= a^0.9 b^0.1
with equality iff a == b
here taking a = u/0.9 and b = v/0.1, we have
u + v >= (u/0.9)^0.9 (v/0.1)^0.1
with equality iff u/0.9 == v/0.1.
Here the best you can do is `u + v == 1` and for that inequality must hold so u = 9v hence you get `v = 0.1` and `u = 0.9`.
You can move `a` as argument of the function with default value `0.9` and use `u = a` and `v = 1 - a` in the solution adding the explanation of why this is the optimal solution.